automation/system_test/prepare.py

```diff
@@ -331,19 +333,46 @@ def _override_mlrun_api_env(self):
             args=["apply", "-f", manifest_file_name],
         )
 
+    def _install_devutilities(self):
+        urlscript = "https://gist.github.com/a51d75fe52e95df617b5dbb983c8e6e1.git"
```

Comment:
This points to a personal repository snippet, please adjust to point to an organizational gist.